gen2_demo.py

```diff
@@ -53,7 +68,10 @@
 else:
     in_w, in_h = map(int, args.cnn_input_size.split('x'))
 
-
+if args.stereo_median_size == 0: median = dai.StereoDepthProperties.MedianFilter.MEDIAN_OFF
```

Comment:
I would make this an `else` condition, as default in case `args.stereo_median_size` would get any value other than predicted (e.g. at some place between argparse check and this check the `args.stereo_median_size` would get changed to invalid value, this item is mutable). 
Other approach would be to add an `else` condition with `raise ValueError`. 
Right now, if we would mutate this value to not predicted one, the `median` value would not be defined at all

```suggestion
if args.stereo_median_size == 3: median = dai.StereoDepthProperties.MedianFilter.KERNEL_3x3
elif args.stereo_median_size == 5: median = dai.StereoDepthProperties.MedianFilter.KERNEL_5x5
elif args.stereo_median_size == 7: median = dai.StereoDepthProperties.MedianFilter.KERNEL_7x7
else: median = dai.StereoDepthProperties.MedianFilter.MEDIAN_OFF
```